examples/fork-join-variable-access/fork_join_variable_access.bal

```diff
@@ -12,47 +12,47 @@ function main(string... args) {
     io:println("[default worker] before fork-join: value of name is [", 
                 name, "] value of era is [", era, "]");
 
-    // Declare the fork-join statement.
+    // Declare the `fork-join` statement.
     fork {
         worker W1 {
-            // Change the value of the integer variable 'i' within the worker W1.
+            // Change the value of the integer variable `i` within the worker W1.
             i = 23;
-            // Change the value of map variable 'm' within the worker W1.
+            // Change the value of map variable `m` within the worker W1.
             m["name"] = "Rajasinghe";
-            // Define a new variable within the worker to send to the join block.
+            // Define a new variable within the worker to send to the `join` block.
             string n = "Colombo";
-            // Send the data to the join block of the fork-join from worker W1.
+            // Send the data to the `join` block of the `fork-join` from worker W1.
             (i, n) -> fork;
         }
 
         worker W2 {
-            // Change the value of string variable 's' within the worker W2.
+            // Change the value of string variable `s` within the worker W2.
             s = "Ballerina";
-            // Change the value of map variable 'm' within the worker W2.
+            // Change the value of map variable `m` within the worker W2.
             m["era"] = "Kandy";
-            // Send the data to the join block of the fork-join from worker W2.
+            // Send the data to the `join` block of the `fork-join` from worker W2.
             s -> fork;
         }
     } join (all) (map results) {
         int p;
         string l;
         // Declare variables to receive the results from the forked workers W1 and W2.
-        // The 'results' map contains a map of any typed values from each worker defined within the fork-join statement
-        // The tuple value received from worker W1 is de-structured and assigned to variables "p" and "l".
+        // The `results` map contains a map of any type values from each worker defined within the `fork-join` statement
```

Comment:
This should be map of `any` typed values

Reply:
Fixed in commit https://github.com/ballerina-platform/ballerina-examples/pull/681/commits/4fe16e8d7e4714dccc3d2d9615c75229f8e28f3c